requirements.txt

```diff
@@ -1,3 +1,5 @@
+jax==0.4.23
```

Comment:
I am not really in favour of a hard pinning of Jax libraries, considering how difficult they are for everyone to match depending on GPU and other issues. From my messing around in a clean conda environment, this was not necessary. Is there any reason you really need to do this? I am on board with a `>=` restriction to bar low versions but I am not a fan of this approach.

Reply:
Do you think we can pin it with ">=" or do we need an upper bound as well?